### ISSUE TITLE
Architecture for gamma,delta

### ISSUE THREAD

**Issue by SrinjaySarkar** [yzheng97/Point-Transformer-Cls#1]
Hi @yzheng97  thank you for the code for point transformer.
The paper mentions that both delta(position encoding) and the gamma(mapping function) are linear functions (MLP with Relu non linearity). Your code designs them as 2d conv layers with batch norm and Relu non linearity. Do you find any significant gain in performance with this change? If no could you please explain the reason behind this ?

**Comment by yzheng97** [comment]
In my trial, no significant gain is observed. I chose to use conv1d/conv2d only because its input/output dimension follows the same [B, C, N1, N2, ....] format with BatchNorm1d/BatchNorm2d in pytorch.

**Comment by yzheng97** [comment]
From my knowledge, there is no big difference between Linear layer and Conv layer in pytorch. As claimed in https://github.com/fxia22/pointnet.pytorch/issues/39#issuecomment-483354156
